packages/htm/README.md

```diff
@@ -18,34 +18,36 @@ npm install @remote-ui/htm --save
 
 ## Usage
 
-`@remote-ui/htm` provides two functions. The first, `htm`, is a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which binds the main function from the `htm` library into a basic object structure. `render`, the second function exported from this library, takes this generated structure and a [`@remote-ui/core` `RemoteRoot`](../core#remoteroot), and appends all of the resulting components into that root.
+`@remote-ui/htm` provides two functions. The first, `createRender`, is a factory function that takes a [`@remote-ui/core` `RemoteRoot`](../core#remoteroot), and returns a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates). This tagged template binds the main function from the `htm` library into creating a tree of `@remote-ui/core` `RemoteComponent` and `RemoteText` objects. The second function, `append`, is a helper to call `appendChild()` on a `RemoteRoot` or `RemoteComponent` for each node in the tree returned by the tagged template.
 
 ```ts
 import {createRemoteRoot} from '@remote-ui/core';
-import {htm, render} from '@remote-ui/htm';
+import {createRender, append} from '@remote-ui/htm';
 
 const root = createRemoteRoot(() => {});
+const ui = createRender(root);
```

Comment:
The `createX(root)` value needs to be the name of the tagged template function — you've moved it to the function that appends, but it needs to be on the tagged template so that it can actually create remote-ui objects.